lib/calyx/version.rb

```diff
@@ -1,3 +1,3 @@
 module Calyx
-  VERSION = '0.5.2'.freeze
+  VERSION = '0.6.2'.freeze
```

Comment:
If it goes up to the next minor version, the patch number can be reset, so `0.6.0`.